### PR TITLE
New package: src-1.40

### PR DESCRIPTION
**Pull request by Eloitor**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by Eloitor** [comment]
I don't know how to fix the tests... 

Reported upstream but the author cannot reproduce: https://gitlab.com/esr/src/-/issues/28

File: srcpkgs/src/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'src'
+pkgname=src
+version=1.37
+revision=1
+build_style=gnu-makefile
+hostmakedepends="python3"
```
Comment (meator) [inline srcpkgs/src/template:6]:
```suggestion
hostmakedepends="python3 rsc"
```

It tries to execute `rcs --version 2>/dev/null` in build and check stages. The check error persists though.

Comment (Eloitor) [reply srcpkgs/src/template]: Wouldn't it be checkdepends?

Comment (meator) [reply srcpkgs/src/template:6]: > Wouldn't it be checkdepends?

It is used in the build stage also.

Comment (Eloitor) [reply srcpkgs/src/template]: ok thanks

**Pull request closed without merging**